### PR TITLE
Grasping Visual Servoing Tweaks

### PR DESCRIPTION
**Pull request by hello-lamsey**
## Description

Adds a loose grasp option for bigger objects to prevent gripper servo overload.
Tunes a few hyperparameters.
Removes attempted grasp at beginning of servoing loop "If there's any chance the object is close enough, we should just try to grasp it." - this was causing missed grabs.

## Checklist

- \[ \] I have performed a self-review of my code
- \[ \] If it is a core feature, I have added thorough tests
- \[ \] I have added documentation for the changes
- \[ \] I have updated the README file if necessary
- \[ \] I have run on hardware if necessary

## Screenshots (if applicable)

Add any relevant screenshots or screen recordings to help reviewers understand the changes.

## Additional context

Add any other context or information about the pull request here.

### CODE REVIEW

File: src/stretch/agent/operations/grasp_object.py [hunk 1]
```diff
@@ -376,13 +378,13 @@ def visual_servo_to_object(
...
                 print("Center in mask?", center_in_mask)
                 print("Current XYZ:", current_xyz)
-                if center_in_mask and (
-                    center_depth < self.median_distance_when_grasping
-                    or median_object_depth < self.median_distance_when_grasping
-                ):
-                    "If there's any chance the object is close enough, we should just try to grasp it." ""
-                    success = self._grasp()
-                    break
+                # if center_in_mask and (
```
Comment (hello-cpaxton) [inline src/stretch/agent/operations/grasp_object.py:381]: Should this just be deleted?

**Comment by hello-cpaxton** [comment]
LGTM; can you share a video of this behavior?

File: src/stretch/agent/operations/grasp_object.py (hunk 1 above)
Comment (hello-lamsey) [reply src/stretch/agent/operations/grasp_object.py:381]: Probably.

**Comment by hello-lamsey** [comment]
footage of two failure modes [here](https://drive.google.com/drive/folders/1fysXnfUgr2WIP1L7ptb8weZ3L1IZakLq?usp=sharing)

This happened more than half of the time that I ran code to grasp the mug / can. For the mug, more than 75% of the time.

**Comment by hello-cpaxton** [comment]
LGTM

**Pull request merged**